### DISCUSSION TITLE
hintTextColor for IOS to change placeholder text color

### DISCUSSION THREAD

**Discussion by amitava82** [software-mansion/react-native-screens#1758] [Ideas]
I see `hintTextColor` works only on Android. When the header has a dark background the placeholder text still shows in grey and there is no way to change the placeholder text color on IOS. it is almost impossible to read the text. Can we have the option to change this please? Is there any temporary workaround?

<img width="336" alt="Screenshot 2023-04-17 at 9 17 22 AM" src="https://user-images.githubusercontent.com/1279530/232373340-68fe8e8a-041b-49f6-8400-9301d62c0570.png">

**Comment by mrzachnugent** [comment]
I found that the placeholder text on iOS is set from the color theme detected from the `Appearance.getColorScheme()`. If the color scheme is light then the placeholder text is dark and vice-versa.

To fix this issue on my end, I used `Appearance.setColorScheme('dark');`

**Comment by numberinho** [comment]
@mrzachnugent  Where did you change `Appearance.setColorScheme('dark');`? Im still struggling on that. Do you have any idea, how to use a custom color in there in iOS?

**Reply by mrzachnugent** [reply]
Last time I checked, I don't think it was possible to add a custom color for iOS.

I set `Appearance.setColorScheme('dark')` on initial load (in a useEffect) or when the user selects dark mode. If that doesn't work, you might need to add https://docs.expo.dev/develop/user-interface/color-themes/ in your project.
